docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
     build: https://github.com/hugsy/ctfpad.git#master
     command: python manage.py runserver 0.0.0.0:8000
     environment:
-      - HEDGEDOC_URL=http://localhost:3000                  # Change here to your server public IP / FQDN
+      - HEDGEDOC_URL=http://hedgedoc:3000                   # Change here to your server public IP / FQDN
```

Comment:
Those variables are not for docker, but the app works. For example `http://hedgedoc:3000` doesn't exist outside the docker network so we can't use that.

Reply:
I understand :) But maybe it would be best for `ctfpad` to just work out of the box so that people can give it a quick try locally before adjusting all the right parameters for a proper deployment. People might just move on to quickly otherwise.

Reply:
Maybe, but that setting would only make it work locally, and the value`http://localhost:3000` for `HEDGEDOC_URL` also does because it is exposed by docker. So drop this change (i.e. just the  change for `HEDGEDOC_URL` ) and I'll merge the other one.

Reply:
oops sorry that is what i meant to do, good catch :heart: